test/apis.js

```diff
@@ -89,13 +89,6 @@ describe('wns.createToast', function () {
 		);
 	});
 
-	it('fails with wrong parameter types', function () {
```

Comment:
I know it's been a while, but do you remember why you removed those tests?

Reply:
After three years, I have absolutely no idea.